creator/ingest_runs/genomic_data_loader.py

```diff
@@ -0,0 +1,397 @@
+import ast
+import os
+import logging
+from pprint import pprint
+
+import pandas as pd
+
+from kf_lib_data_ingest.app import settings as ingest_settings
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from kf_lib_data_ingest.common.constants import GENOMIC_FILE
+from kf_lib_data_ingest.etl.load.load import LoadStage
+
+from creator.ingest_runs import utils
+
+FILE_EXT_FORMAT_MAP = {
```

Comment:
This should probably be put in a common module that this module and the s3_scrape_config.py can both access. Write now the same structure is defined in two diff places. Could maybe go in creator/ingest_runs/utils.py?